frontend/src/concepts/topology/PipelineDefaultTaskGroup.tsx

```diff
@@ -0,0 +1,139 @@
+import * as React from 'react';
+
+import {
+  Dimensions,
+  LabelPosition,
+  OnSelect,
+  WithSelectionProps,
+  getEdgesFromNodes,
+  getSpacerNodes,
+  isNode,
+  DefaultTaskGroup,
+  observer,
+  Node,
+  GraphElement,
+  action,
+} from '@patternfly/react-topology';
+import PipelineTaskGroupCollapsed from './PipelineTaskGroupCollapsed';
+import { NODE_HEIGHT, NODE_WIDTH } from './const';
+
+export interface EdgeCreationTypes {
+  spacerNodeType?: string;
+  edgeType?: string;
+  spacerEdgeType?: string;
+  finallyNodeTypes?: string[];
+  finallyEdgeType?: string;
+}
+
+interface PipelinesDefaultGroupProps {
+  children?: React.ReactNode;
+  className?: string;
+  element: GraphElement;
+  hover?: boolean;
+  collapsible?: boolean;
+  collapsedWidth?: number;
+  collapsedHeight?: number;
+  onCollapseChange?: (group: Node, collapsed: boolean) => void;
+  selected?: boolean;
+  onSelect?: OnSelect;
+  recreateLayoutOnCollapseChange?: boolean;
+  getEdgeCreationTypes?: () => {
+    spacerNodeType?: string;
+    edgeType?: string;
+    spacerEdgeType?: string;
+    finallyNodeTypes?: string[];
+    finallyEdgeType?: string;
+  };
+}
+
+type PipelinesDefaultGroupInnerProps = Omit<PipelinesDefaultGroupProps, 'element'> & {
+  element: Node;
+} & WithSelectionProps;
+
+const DefaultTaskGroupInner: React.FunctionComponent<PipelinesDefaultGroupInnerProps> = observer(
+  ({ className, element, getEdgeCreationTypes, selected, onSelect, ...rest }) => {
+    const childCount = element.getAllNodeChildren().length;
+    const data = element.getData();
+
+    const handleCollapse = action((group: Node, collapsed: boolean): void => {
```

Comment:
This will get done in the DefaultTaskGroup for expanded groups. Remove this here and move it to `PipelineTaskGroupCollapsed`.